tests/test_sso.py

```diff
@@ -354,6 +354,42 @@ def test_list_connections(
 
         assert connections_response["data"] == mock_connections["data"]
 
+    def test_list_connections_with_connection_type_as_invalid_string(
+        self, setup_with_client_id, mock_connections, mock_request_method
+    ):
+        mock_request_method("get", mock_connections, 200)
+
+        try:
+            self.sso.list_connections(connection_type="UnknownSAML")
+        except Exception as ex:
+            assert str(ex) == "'connection_type' must be a member of ConnectionType"
```

Comment:
Can we use `pytest.raises`, and also assert on a more specific error type?

```suggestion
        with pytest.raises(
            ValueError, match="'connection_type' must be a member of ConnectionType"
        ):
            self.sso.list_connections(connection_type="UnknownSAML")
```